src/components/segments/MeetupDate.js

```diff
@@ -4,41 +4,41 @@ import moment from 'moment';
 
 class MeetupDate extends React.Component {
 
-	render() {
+  render() {
 
-		return this.props.loading ? (
-			<div className="interested-calendar">
+    return (
+      <div className="interested-calendar">
         <div className="flexgrid-row">
           <div className="calendar-date">
-            <div className="month">Month</div>
-            <div className="day">00</div>
+            <div className="month">{this.props.date.format('MMMM')}</div>
+            <div className="day">{this.props.date.format('D')}</div>
           </div>
           <div className="calendar-text">
-            <a href="#" target="_blank" className="meetup-btn">Join others at our next Project Night</a>
+            <a href={this.props.event_url} target="_blank" className="meetup-btn">Join {this.props.rsvps} others at our next event!</a>
+            <p className="date">{this.props.name}</p>
             <p className="date">
+              {this.props.date.format('dddd, MMMM Do, YYYY')}
               <br />6:00 PM to 9:00PM
             </p>
           </div>
         </div>
       </div>
-		)	: (
-			<div className="interested-calendar">
-        <div className="flexgrid-row">
-          <div className="calendar-date">
-            <div className="month">{this.props.meetup.start_time.format('MMMM')}</div>
-            <div className="day">{this.props.meetup.start_time.format('D')}</div>
-          </div>
-          <div className="calendar-text">
-            <a href={this.props.meetup.event_url} target="_blank" className="meetup-btn">Join {this.props.meetup.rsvps} others at our next Project Night</a>
-            <p className="date">
-            	{this.props.meetup.start_time.format('dddd, MMMM Do, YYYY')}
-              <br />6:00 PM to 9:00PM
-            </p>
-          </div>
-        </div>
-      </div>
-		)
-	}
+    )
+  }
 }
 
+MeetupDate.propTypes = {
```

Comment:
I'm pretty sure we have babel configured to transpile [`static` properties](https://github.com/jeffmo/es-class-fields-and-static-properties). I personally find that a bit more clean because you can define it at the top of the class e.g.

```js
class MeetupDate extends React.Component {
  static propTypes = {
    ...
  };
  
  ...
}
```